reference/forms/types/options/by_reference.rst.inc

```diff
@@ -31,7 +31,7 @@ Notice that ``setAuthor()`` is not called. The author is modified by reference.
 If you set ``by_reference`` to false, submitting looks like this::
 
     $article->setTitle('...');
-    $author = $article->getAuthor();
+    $author = (clone) $article->getAuthor();
```

Comment:
Very minor, but `clone` isn't a type cast so please remove the `()` to avoid confusion.